patches/notify-add-snap-parameters.patch

```diff
@@ -0,0 +1,20 @@
+--- a/notification-scripts/notify	2020-06-18 21:40:02.000000000 +0200
++++ b/notification-scripts/notify	2020-06-25 01:27:28.997695247 +0200
+@@ -25,7 +25,7 @@
+ notify_CATEGORY="im.received"
+ 
+ # Notification header
+-notify_HEAD="Matterhorn message from $sender"
++notify_HEAD="Message from $sender"
+ 
+ # Escape backslashes in the message
+ message="${message//\\/\\\\}"
+@@ -62,5 +62,8 @@
+ if [ ! -z "$urgency" ]
+ then
+     test "$urgency" = "none" ||
++        notify-send -i $SNAP/meta/gui/${SNAP_INSTANCE_NAME}.png \
++            -h "string:desktop-entry:${SNAP_NAME}_${SNAP_INSTANCE_NAME}" \
```

Comment:
This is probably only doing what you want by coincidence.  `SNAP_INSTANCE_NAME` is not the app within the snap: instead it is the unique package name when you parallel install two copies of a snap (a feature which still isn't enabled by default).

I don't think there is an environment variable representing what you want, so perhaps just list the desktop ID explicitly?

Reply:
Thanks, fixed.